clap_complete_fig/Cargo.toml

```diff
@@ -1,7 +1,9 @@
+cargo-features = ["workspace-inheritance"]
+
 [package]
 name = "clap_complete_fig"
 version = "3.1.5"
-edition = "2018"
+edition.workspace = true
 include = [
```

Comment:
We actually should use the same include for everything.

Reply:
There was a few that were different so I couldn't tell. I'll make them all inherit from a workspace

Reply:
Skew from forgetting to update which is why this feature is helpful!